tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.jobs/Functions.cs

```diff
@@ -447,7 +447,7 @@ private static string TeamMembersPermissionInformation(MatterDetails matterDetai
                                         log.WriteLine($"The user {email} has been present in the system and he has accepted the invitation and providing permssions to  matter {originalMatter.Matter.Name} from the user {email}");
                                         UpdateMatter umd = new UpdateMatter();
                                         //Update all matter related lists and libraries permissions for external users
-                                        if (originalMatter.MatterUpdateStatus.ToLower() == "pending")
+                                        if (azureTableMatterInformationVMRow.MatterUpdateStatus.ToLower() == "pending")

```

Comment:
Need to check the status from the azure table column instead of checking in the status in the serialized data